src/agents/gc/OSConfig_Proto.schema.mof

```diff
@@ -0,0 +1,71 @@
+#pragma include ("cim_schema_2.26.0.mof")
+#pragma include ("MSFT_Qualifiers.mof")
+#pragma include ("OMI_BaseResource.mof")
+#pragma include ("MSFT_Credential.mof")

```

Comment:
you are not using this class? can we remove it

Reply:
It was generated with the rest of the NRP code. It is invoked into the build.